settings.lua

```diff
@@ -2619,7 +2619,7 @@ function StrawOnHeadland:init()
 	BooleanSetting.init(self, 'strawOnHeadland', 'COURSEPLAY_STRAW_ON_HEADLAND',
 				'COURSEPLAY_YES_NO_STRAW_ON_HEADLAND', nil)
 	-- set default while we are transitioning from the the old setting to this new one
-	self:set(false)
+	self:set(true)
```

Comment:
I'm not sure if I'm overthinking this, but I believe current Courseplay drops on headlands, therefore, we want to default to true?

Reply:
Yes that is ok. I didn't think about this when I added this setting :)